docs/whirlpool/docs/03-Whirlpools SDKs/01-Whirlpools/06-Send Transaction.mdx

```diff
@@ -67,133 +72,17 @@ Let's start by importing the necessary dependencies.
   </TabItem>
 </Tabs>
 
+### 2. Create Transaction Message From Instructions
+To send a transaction on Solana, you need to include a blockhash to the transaction. A blockhash acts as a timestamp and ensures the transaction has a limited lifetime. Validators use the blockhash to verify the recency of a transaction before including it in a block. A transaction referencing a blockhash is only valid for 150 blocks (~1-2 minutes, depending on slot time). After that, the blockhash expires, and the transaction will be rejected.
```

Comment:
You can also sign using a durable nonce. You could add a small note that that is also possible but not what we are using in this example